package/MDAnalysis/transformations/positionaveraging.py

```diff
@@ -0,0 +1,102 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+
+
+"""
+Trajectory Coordinate Averaging --- 
```

Comment:
remove `---`

Reply:
make sure that the docs are integrated into the docs under [doc/sphinx/source/documentation_pages/transformations](https://github.com/MDAnalysis/mdanalysis/tree/develop/package/doc/sphinx/source/documentation_pages/transformations)

Please build the docs locally and check, see https://github.com/MDAnalysis/mdanalysis/wiki/WritingDocumentation#generating-the-docs